Documentation/joss/paper.md

```diff
@@ -19,50 +9,85 @@ authors:
     orcid: 0000-0003-0387-3440
     affiliation: 1
   - name: Helge Hecht
-    affiliation: 2
+    affiliation: 1
   - name: Julian Amann
     affiliation: 1
   - name: Jonas Schlenger
+    orcid: 0000-0002-4875-5117
     affiliation: 1
 affiliations:
  - name: Chair for Computational Modelling and Simulation, Technical University of Munich
    index: 1
- - name: ??
-   index: 2
 date: 24 December 2020
 bibliography: paper.bib
 ---
 
 # Summary
 
-> TODO: A summary describing the high-level functionality and purpose of the software for a diverse, *non-specialist* audience.
+*TUM Open Infra Platform* (OIP) is an open source application for
+ viewing and analysis of different BIM models used in the civil engineering field.
+OIP supports reading, visualization, navigating, and handling of:
 
-TUM OpenInfraPlatform (OIP) is a Software-as-a-Service providing an open source application for
- viewing and analysis of different digital models used in the civil engineering field.
+- Industry Foundation Classes (IFC) models as specified in ISO 16739 [@iso16739];
+- Point Cloud Data (PCD) models as supported by the Cloud Compare library [CloudCompare].
 
-OIP supports reading and visualization of:
+Multiple models can be loaded at once and compared between each other (see Figure \ref{fig:ifcpcd}).
+Their absolute position is accounted for, so the models can be checked against one another for internal differences based on location of elements.
 
-- Industry Foundation Classes (IFC) models as specified in ISO 16739
-- Point Cloud Data (PCD) models as supported by the Cloud Compare library
+![A PCD model together with an IFC model loaded in OIP.\label{fig:ifcpcd}](../images/ifc_and_pcd.png)
 
-Multiple models can be loaded at once and compared between each other.
+Additionally, OIP incorporates its own EXPRESS lexer and parser.
+This enables automatic code generation for IFC early-binding library [@Hecht:2019:FBI].
+As such, complete contents of IFC files can be interpreted and analysed.
+Moreover, type safety is guaranteed at compile time thus reducing the risk for bugs and errors.
 
-Additionally, OIP incorporates its own EXPRESS parser to automatically generate early-binding code for IFC [Hecht:2019:FBI].
+Let it be stated here that OIP serves as a prototypical playground for developments.
+During the past 3 years, OIP was completely restructured and its code base overhauled - the IFC now stands as the internal model [@Hecht:2019:FBI].
```

Comment:
I would delete this sentence: ~~During the past 3 years, OIP was completely restructured and its code base overhauled~~ write -> "OIP uses as an internal data model the IFC Schema" (not sure how you handle different versions IFC2x3, IFC4 or if you support only the newest and gretes IFC4x3 ;))

Reply:
resolved in 327d66e20deef940696b0540e3ee53d9996b1643